source/documentation/getting_started/quick_start.md

```diff
@@ -10,7 +10,7 @@ Your API key can be used for all registers.
 
 You should include your key in all requests using the `Authorization` header:
 
-`curl https://country.register.gov.uk/record/GB.json --header "Authorization: YOUR-API-KEY-HERE"`
+`curl https://country.register.gov.uk/record/GB.json --header 'Authorization: YOUR-API-KEY-HERE'`
```

Comment:
this should be `/records/` now.

Reply:
@gidsg is correct, though I think (fortunately) this is the only example where this is incorrect

Reply:
ah good spot!